src/common/transformations/src/transformations/common_optimizations/lin_op_sequence_fusion.cpp

```diff
@@ -87,6 +87,9 @@ ov::pass::AddAddFusion::AddAddFusion() {
         auto add1 = label_to_output[m_add1].get_node_shared_ptr();
         auto add2 = label_to_output[m_add2].get_node_shared_ptr();
 
+        if (!add1->has_evaluate() || !add2->has_evaluate())
+            return false;
+
```

Comment:
btw, I think the same problem exists for other similar transformations

Reply:
can we use another folding function instead of `op::util::eltwise_fold`?
`op::util::eltwise_fold` throws exception
probably `ov::util::get_constant_from_source`,  `make_try_fold`?

Reply:
Multiply ov::op::v1::Multiply supports f64. There is already test MulMulMulFusion_f64, that checks it. I also checked AddMultiplyFusion - it works with f64 input

Reply:
> can we use another folding function instead of `op::util::eltwise_fold`? `op::util::eltwise_fold` throws exception probably `ov::util::get_constant_from_source` or `make_try_fold`?

I fixed it now

Reply:
I would recommend to use `make_try_fold` instead of eltwise_fold everywhere in transformations for ops with unary output.

Reply:
created separate PR https://github.com/openvinotoolkit/openvino/pull/29295 for it